### PR TITLE
update debian packaging

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -10,7 +10,7 @@ Build-Depends: debhelper (>= 9), cmake,
  doxygen, texlive-latex-extra, texlive-latex-recommended, ghostscript,
  libboost-filesystem-dev, libboost-system-dev, libboost-date-time-dev,
  libboost-test-dev, libsuperlu3-dev, gfortran, libsuitesparse-dev, libert.ecl-dev,
- libopm-parser-dev, libboost-iostreams-dev
+ libopm-parser-dev, libboost-iostreams-dev, libopm-common-dev
```
Comment (atgeirr) [inline debian/control]: Does this not depend on opm-material?

Comment (akva2) [inline debian/control]: yes it does, and it always did (or at least it did in 2015.04)

Comment (atgeirr) [inline debian/control]: Again, sorry for missing that it was already there (misunderstood order etc.)

**Comment by andlaus** [comment]
seems fine. thanks for your work!

**Pull request merged**